hugo.yaml

```
@@ -34,13 +34,13 @@ menu:
         teaser: >
           Discover our recommendations for developing benchmarks.
 
-    #- name: Projects
-      #pageRef: /projects
-      #weight: 3
-      #params:
-        #type: playbook
-        #teaser: >
-          #Discover our recommendations for project management.
+    - name: Projects
+      pageRef: /projects
```

```suggestion
      pageRef: /project
```

either that, or change the directory name to `projects` (`git mv content/project content/projects`)

---

Do you have a preference on whether it's called the Project Playbook or the Projects Playbook?